README.md

```diff
@@ -1 +1,37 @@
-# megafuel-js-sdk
\ No newline at end of file
+# megafuel-js-sdk
+
+This is Javascript SDK for the MegaFuel clients, it provides a simple way to interact with the MegaFuel.
```

Comment:
This JavaScript SDK is thin wrapper of MegaFuel clients, offering a streamlined interface to interact with the [MegaFuel](https://docs.nodereal.io/docs/megafuel-overview).

Reply:
done